.travis.yml

```diff
@@ -22,10 +22,6 @@ install:
   - npm config set spin false
   - npm install
 
-before_script:
-  - bash bin/download-firefox-travis.sh release-linux64-add-on-devel
```

Comment:
probably can remove `bin/download-firefox-travis.sh` now or is it used elsewhere?

Reply:
yeah I was just trying to see what happens. Wasn't sure if `npm <cmd>` would pick up the system firefox since our previous script specifically exported it.